extensions-core/hdfs-storage/pom.xml

```diff
@@ -255,7 +255,6 @@
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-aws</artifactId>
           <version>${hadoop.compile.version}</version>
-          <scope>provided</scope>
```

Comment:
The dependency analysis job is flagging this. I suggest changing the scope to `runtime`.